pages/trip/[id].js

```diff
@@ -18,6 +19,15 @@ export default function TripDetails() {
   if (!trips || isLoading) {
     return <>is Loading...</>;
   }
+
+  async function handleDelete() {
+    console.log("TEST");
+    await fetch(`/api/trips/${id}`, {
+      method: "DELETE",
+    });
+    router.push("/");
+  }
+
```

Comment:
Currently, you are always redirecting the user back to the homepage, no matter if the deletion was successful or not. You could for example implement error handling with a `try/catch` block.